Cargo.toml

```
@@ -27,7 +27,7 @@ unicode-segmentation = "1.2"
 unicode-width = "0.1"
 termion = { version = "1.5" , optional = true }
 rustbox = { version = "0.11", optional = true }
-crossterm = { git = "https://github.com/crossterm-rs/crossterm", optional = true }
+crossterm = { version = "0.17.3", optional = true }
```

```suggestion
crossterm = { version = "0.17", optional = true }
```
I tend to put more flexible version constraints so that end users get bug fixes release without a `tui` release.